Tests/Packet++Test/Tests/GtpTests.cpp

```diff
@@ -282,3 +283,356 @@ PTF_TEST_CASE(GtpLayerEditTest)
 
 	delete[] buffer2;
 }  // GtpLayerEditTest
+
+PTF_TEST_CASE(GtpV2LayerParsingTest)
+{
+	timeval time{};
+	gettimeofday(&time, nullptr);
+
+	{
+		READ_FILE_AND_CREATE_PACKET(1, "PacketExamples/gtpv2-with-teid.dat");
+		pcpp::Packet gtpPacket(&rawPacket1);
+
+		PTF_ASSERT_TRUE(gtpPacket.isPacketOfType(pcpp::GTPv2));
+		PTF_ASSERT_TRUE(gtpPacket.isPacketOfType(pcpp::GTP));
+		auto gtpLayer = gtpPacket.getLayerOfType<pcpp::GtpV2Layer>();
+		PTF_ASSERT_NOT_NULL(gtpLayer);
+
+		PTF_ASSERT_EQUAL(gtpLayer->getMessageType(), pcpp::GtpV2MessageType::ModifyBearerRequest);
+		PTF_ASSERT_EQUAL(gtpLayer->getMessageLength(), 107);
+		PTF_ASSERT_FALSE(gtpLayer->isPiggybacking());
+		PTF_ASSERT_EQUAL(gtpLayer->getHeaderLen(), 111);
+		auto teid = gtpLayer->getTeid();
+		PTF_ASSERT_TRUE(teid.first);
+		PTF_ASSERT_EQUAL(teid.second, 0xd37d1590);
+		PTF_ASSERT_EQUAL(gtpLayer->getSequenceNumber(), 0x1a4a43);
+		PTF_ASSERT_FALSE(gtpLayer->getMessagePriority().first);
+		PTF_ASSERT_EQUAL(gtpLayer->toString(), "GTPv2 Layer, Modify Bearer Request message");
+		PTF_ASSERT_NULL(gtpLayer->getNextLayer());
+
+		PTF_ASSERT_EQUAL(gtpLayer->getInformationElementCount(), 9);
+
+		// clang-format off
+		auto expectedIEValues = std::vector<std::tuple<pcpp::GtpV2InformationElement::Type, size_t, size_t, uint8_t, uint8_t, uint8_t, uint8_t>> {
+			std::tuple<pcpp::GtpV2InformationElement::Type, size_t, size_t, uint8_t, uint8_t, uint8_t, uint8_t>{ pcpp::GtpV2InformationElement::Type::Uli,            17, 13, 0, 0, 0x18, 0x64 },
```

Comment:
Consider giveing this `std::tuple<pcpp::GtpV2InformationElement::Type, size_t, size_t, uint8_t, uint8_t, uint8_t, uint8_t>` a shorthand.

Reply:
Fixed in https://github.com/seladb/PcapPlusPlus/pull/1634/commits/4f24ebdb51c22c504f5375972c9710c777915d1b